zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -126,8 +134,8 @@
                           <text macro="volume"/>
                         </group>
                         <names variable="editor">
+                          <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
                           <label form="verb-short" suffix=" "/>
```

Comment:
I did not change this either. I only inserted:
and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"